backend/src/apiserver/client_manager.go

```diff
@@ -48,7 +48,7 @@ const (
 	mysqlServicePort      = "MYSQL_SERVICE_PORT"
 
 	podNamespace          = "POD_NAMESPACE"
-	dbName                = "mlpipeline"
+	dbName                = "DBConfig.dbName"
```

Comment:
`DBConfig.DBName` to be consistent. You can also name this const `mysqlDBName` instead of `dbName`

Please add the default to https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/config/config.json as well